### DISCUSSION TITLE
无法申请学生权益

### DISCUSSION THREAD

**Discussion by DaydreamerCCHUST** [community/community#147308] [GitHub Education]
### Hi everyone,

我上传了我的学生证，但是学生权益被拒绝了，然后我又上传了学信网的学籍也被拒绝了，该怎么办呀？

**Comment by Logophoman** [comment]
Translation for anyone Wondering: 

"I uploaded my student ID, but my student rights were rejected, then I uploaded my school records and they were also rejected, what should I do?"

Unfortunately I can't help with that...
